assets/airports/airportLoadList.json

```diff
@@ -373,6 +373,12 @@
         "name": "San Antonio International Airport",
         "premium": false
     },
+    {
+        "icao": "ksav",
+        "level": "easy",
```

Comment:
With the currently-defined traffic levels, this falls under the 'medium' category (40-60 acph). Either reduce the traffic to be within 20-40 acph or change the level to 'medium'

Reply:
Fixed